pyneuroml/utils/__init__.py

```diff
@@ -473,6 +473,61 @@ def rotate_cell(
 
     return newcell
 
+def translate_cell_to_origin(
+    cell: neuroml.Cell,
+    inplace: bool = False,
+) -> neuroml.Cell:
+    """Return a new cell object translated so that the some is at the origin.
```

Comment:
"Translate cell to origin" is probably a better description. the `inplace` param tells users that they can use the same object if they wish. What do you think?

Reply:
That's true, we should probably change that at the rotate_cell() function as well